package.json

```diff
@@ -37,8 +37,11 @@
   "dependencies": {
     "basekick": "^3.0.0",
     "classnames": "^2.2.6",
-    "prop-types": "^15.6.2",
-    "react": "^16.5.2"
+    "prop-types": "^15.6.2"
+  },
+  "peerDependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
```

Comment:
I don't think `react-dom` is a peer dependency. We only import `react` in our code.